sources/packages/api/src/route-controllers/designation-agreement/designation-agreement.aest.controller.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  Controller,
+  Get,
+  Param,
+  NotFoundException,
+  Patch,
+  Body,
+  Query,
+  BadRequestException,
+} from "@nestjs/common";
+import { DesignationAgreementService, FormService } from "../../services";
+import { FormNames } from "../../services/form/constants";
+import { DesignationAgreementStatus } from "../../database/entities";
+import { getISODateOnlyString, PaginationParams } from "../../utilities";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups, UserToken } from "../../auth/decorators";
+import { IUserToken } from "../../auth/userToken.interface";
+import { UserGroups } from "../../auth/user-groups.enum";
+import {
+  GetDesignationAgreementDto,
+  GetDesignationAgreementsDto,
+  PendingDesignationDto,
+  UpdateDesignationDto,
+} from "./models/designation-agreement.model";
+import { DesignationAgreementServiceController } from "./designation-agreement.service.controller";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("designation-agreement")
+export class DesignationAgreementAESTController {
+  constructor(
+    private readonly designationAgreementServiceController: DesignationAgreementServiceController,
```

Comment:
I believe that the naming should end with what the class represent, in this case a service (e.g. _designationAgreementControllerService_).

Reply:
Service controller renamed to controller service.